bad_actors/flask_api/flask_api.py

```diff
@@ -0,0 +1,80 @@
+from flask import Flask, jsonify, abort, make_response, request
+import sys
+import json
+from flask_sqlalchemy import SQLAlchemy
+
+sys.path.append("C:\Users\leahk\Desktop\measurement-of-online-discussion-authenticity\\bad_actors")
+from configuration.config_class import getConfig
+from validate_json import validate_json
+
+
+# config
+app = Flask(__name__)
+#config = getConfig()
+#json_file = config.get('Schema', 'json_file')
+json_file = 'configuration/config_api.json'
+with open(json_file, 'r') as f:
+    json_config = json.load(f)
+
+schema = json_config['schema']
+print schema
+test_data = [item for item in json_config['test_data']]
```

Comment:
I see test_data is not used in this file, so I don't think it has to be here. test_data can be used for the unit-testing